README.md

```diff
@@ -50,21 +50,21 @@ First-timers-bot is built with [Probot](https://probot.github.io/).
 
 ### Configuration
 
-The first-timers app works without configuration. If you want to change the default settings, create a .github/first-timers.yml file with the content below and then you can adjust the options to your preference.
+The first-timers app works without configuration. If you want to change the default settings, create a .github/first-timers.yml file with the content below and then you can adjust the options to your preference.  When both the template & the repository is set, then the template is loaded from the _configured_ repository at the configured template path.
```

Comment:
This is great :)

I wonder if we could link to Hoodie’s setup as an example? E.g. this repositorie’s [.github/first-timers.yml file](https://github.com/hoodiehq/first-timers-bot/blob/master/.github/first-timers.yml) is using the [.github/first-timers.yml file on the hoodiehq/camp repository](https://github.com/hoodiehq/hoodie/blob/master/.github/first-timers.yml) to create an issue like this one: https://github.com/hoodiehq/camp/issues/126